packages/communication-react/review/communication-react.api.md

```diff
@@ -1218,7 +1218,7 @@ export const GridLayout: (props: GridLayoutProps) => JSX.Element;
 export interface GridLayoutProps {
     // (undocumented)
     children: React_2.ReactNode;
-    styles?: BaseCustomStyles;
+    styles?: GridLayoutStyles;
```

Comment:
I guess GridLayoutStyles is like a super set of BaseCustomStyles, so this is not a breaking change? If so, let's make this line post-GA because api has been frozen, especially for changes could be resolved after GA

Reply:
Yeah we can make this post GA since it's not a breaking change. Will revert.